Cargo.toml

```diff
@@ -1,28 +1,24 @@
 [workspace]
 resolver = "2"
 members = [
-    "gateway/crates/*",
-    "cli/crates/*",
-    "engine/crates/*",
-    "engine/crates/engine-v2/auth",
-    "engine/crates/engine-v2/config",
-    "engine/crates/engine-v2/axum",
-    "engine/crates/engine-v2/codegen",
-    "engine/crates/engine-v2/walker",
-    "engine/crates/engine-v2/schema",
-    "engine/crates/engine-v2/query-solver",
-    "engine/crates/engine-v2/id-derives",
-    "engine/crates/engine-v2/id-newtypes",
-    "engine/crates/engine-v2/parser",
-    "engine/crates/engine-v2/value",
-    "graph-ref",
-    "graphql-lint",
-    "gqlint",
-    "workspace-hack",
+    "cli",
+    "gateway",
+    "crates/*",
+    "crates/engine-v2/auth",
```

Comment:
can we rename engine-v2 to just engine?

Reply:
That's going to make a huge diff, so I'd prefer to do that as a separate PR.